kodilla-patterns/src/test/java/com/kodilla/patterns/singleton/LoggerTestSuite.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.patterns.singleton;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class LoggerTestSuite {
+    @Test
+    public void loggerTest() {
+        //Given
+        String dateAndTimeOfLogEntry = LocalDate.now() + " " + LocalTime.now();
```

Comment:
Nie jestem pewien po co jest dateAndTimeOfLogEntry. Chodzilo Ci o cos takiego:
LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")) ?
https://www.mkyong.com/java8/java-8-how-to-format-localdatetime/

Mozesz zmodyfikowac logger zeby dodawal czas w metodzie Logger.log